packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -67,6 +69,12 @@ const add = createAction(ADD, nodeMap => ({nodeMap}));
  */
 const merge = createAction(MERGE, nodeMap => ({nodeMap}));
 
+/**
+ * Specific action for node information coming from the guest frame,
+ * to be able to trigger more specific sagas based on that information.
```

Comment:
> "to be able to trigger more specific sagas"
These ain't cool. I'd rather fire an additional action to inform about that, or listen to one of the existing actions, if there are any

Reply:
Mmm, maybe my explanation is bad? So far there is only merge and add which is very generic. We don't know where it comes from. In theory we could also use those BUT as we trigger a merge after we fetched policies that might result in a possible recursion? That's why I added this more specific action.